### ISSUE TITLE
uncertainty

### ISSUE THREAD

**Issue by TsspGit** [TsspGit/CodVid19_CIEMAT#31]
- [x] Train 1 autoencoder + dropout on the test.

- [x] Apply it to RF/XGBoost
